packages/gatsby-theme-aio/src/components/Table/index.js

```diff
@@ -114,6 +114,9 @@ const Tr = ({ children, ...props }) => {
         ? children
         : children.map((child, index) => {
             child.props.cellWidth = columnWidthDistribution[index];
+            if (child.props?.children?.props?.children?.props) {
```

Comment:
Assuming this is for a specifically for the Image componenet: 
Can this be more specific? `child.props?.children?.props?.children?.props` can flag potentially any child component props, and the isTable prop could cause unwanted changes in them.
If we want to set context across all children within table we should set context instead of prop drilling.

Reply:
https://medium.com/geekculture/props-drilling-v-s-context-api-which-one-is-the-best-75c503d21a65